terraform/mgmt/files/deployer_policy.json

```diff
@@ -23,8 +23,7 @@
         "route53:*",
         "route53domains:*",
         "s3:*",
-        "sns:ListSubscriptionsByTopic",
-        "sns:ListTopics"
+        "sns:*"
```

Comment:
I've said it before: we'll probably want to revisit this policy with @ManojChalise, but just made this change to get the deploy working.

Reply:
Looks like its privilege for role. We probably want to give specific permission rather then *. 

General best practices is to follow, theory of least privilege . Meaning granting only permission this that is needed to execute this particular  task .

Reply:
Looks like its privilege for role. We probably want to give specific permission rather then *.

General best practices is to follow, theory of least privilege . Meaning granting only permission this that is needed to execute this particular task .

Reply:
> Looks like its privilege for role.

Yep, this policy applies to Jenkins (for the moment) and CircleCI.

> We probably want to give specific permission rather then *.

I hear you, though I would argue there are diminishing returns in spending time whittling down the policy. Having an overly restrictive policy leads to bugs further down the road, where it's unclear why resources aren't able to talk to one another. Hoping for a baseline deployer policy that is a compromise between least privilege and practicality.

Reply:
+1 to what Manoj said. Now is the time to restrict it as far as possible while still maintaining usability. Any item with *. will definitely be a red flag on an assessment.

Reply:
Cool, we'll do that as a follow-up.

https://trello.com/c/Ikxh4nYk/260-devsecops-example-as-security-i-want-least-privilege-access-for-the-deployer-user